google-cloud-video_intelligence/google-cloud-video_intelligence.gemspec

```diff
@@ -3,7 +3,7 @@
 
 Gem::Specification.new do |gem|
   gem.name          = "google-cloud-video_intelligence"
-  gem.version       = "0.23.0"
+  gem.version       = "0.24.0"
```

Comment:
The [current release](https://github.com/GoogleCloudPlatform/google-cloud-ruby/releases/tag/google-cloud-video_intelligence%2Fv0.24.1) is already `0.24.1`. This should be bumped up to `0.25.0`.